wdae/wdae/gene_scores/views.py

```diff
@@ -106,6 +108,7 @@ def post(self, request):
             score_max = float("inf")
```

Comment:
I would suggest using `score.min()` and `score.max()` instead of `float("inf")` when the min/max is not convertible to float. 

And there are no tests to cover this case.

Reply:
It is OK. But add a test to cover this case.